config.js

```diff
@@ -16,11 +16,11 @@ const ConfigOptions = () => {
     let config = {
         isProduction,
         isDev,
-
         webRootDir: '.',
         assetSrcDir: 'src',
         assetDistDir: 'dist',
         applyRevision: true,
+        packageVersion: consumingPackageConfig && consumingPackageConfig.version,
```

Comment:
More of a question than a comment for change – does this return the value in `consumingPackageConfig.version`?  Looks like it would return a Boolean, but guessing this is a shortcut to test for the config object being present?

Reply:
It's a shorthand falsy check, it should return `undefined` if `version` isn't defined I think.

Reply:
It's probably OK to just use `consumingPackageConfig.version` here, we're not doing the falsy check anywhere else in the module.

Reply:
It will return the `version` property if it's defined otherwise it returns `undefined`.  Agree, I'll update it to `consumingPackageConfig.version`.